cc.newspring.Apollos/Utilities/NewtonsoftJsonSerializer.cs

```diff
@@ -27,7 +27,21 @@ public NewtonsoftJsonSerializer( ISerializer serializer )
 
         public string Serialize( object obj )
         {
-            var json = Newtonsoft.Json.JsonConvert.SerializeObject( obj );
+            var json = string.Empty;
+            var group = obj as Rock.Model.Group;
+
+            if ( group != null )
+            {
+                var temp = group.GroupType;
+                group.GroupType = null;
+                json = Newtonsoft.Json.JsonConvert.SerializeObject( group );
```

Comment:
The serializer was complaining about circular references.  Something like this is problematic for the serializer because it is infinite: group.GroupType.Groups[n] == group